styles.css

```diff
@@ -315,6 +331,118 @@ menu {
   background: #008552;
 }
 
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  display: none;
+  height: 100vh;
+  background-color: #c1c7d0;
+  z-index: 9999;
+}
+
+.modal_item {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 100%;
+  height: 100%;
+  align-items: flex-start;
+}
+
+.modal_img_container {
+  position: relative;
+  width: 100%;
+}
+
+.modal_item img {
+  background: #ebf0ee;
+  height: 252px;
+  box-sizing: border-box;
+}
+
+.anchor {
+  text-decoration: none;
+  color: inherit;
+}
+
+.close_icon {
+  display: flex;
+  box-sizing: border-box;
+  justify-content: end;
+  padding: 0 10px;
+  font-size: 1.5rem;
+  color: #fff;
+  background: #67798e;
+  width: 2rem;
+  position: absolute;
+  top: 30px;
+  right: 25px;
+  z-index: 1000;
+}
```

Comment:
Kindly ensure that your close button has the same background and position as the figma design for both mobile and desktop to match the design. :wink: